docs/concepts/registries.md

```diff
@@ -199,22 +199,22 @@ Creating Custom Forge Registries
 
 Custom registries can usually just be a simple map of key to value. This is a common style; however, it forces a hard dependency on the registry being present. It also requires that any data that needs to be synced between sides must be done manually. Custom Forge Registries provide a simple alternative for creating soft dependents along with better management and automatic syncing between sides (unless told otherwise). Since the objects also use a Forge registry, registration becomes standardized in the same way.
 
-Custom Forge Registries are created via `RegistryBuilder` via either `RegistryEvent$NewRegistry` or the `DeferredRegister`. The class `RegistryBuilder` takes certain parameters (such as the name, the `Class` of its values, and various callbacks for different events happening on the registry). Calling `RegistryBuilder#create` will result in the registry being built, registered to the `RegistryManager`, and returned to the caller for additional processing.
+Custom Forge Registries are created via `RegistryBuilder` via either `NewRegistryEvent` or the `DeferredRegister`. The class `RegistryBuilder` takes certain parameters (such as the name, the `Class` of its values, and various callbacks for different events happening on the registry). Calling `RegistryBuilder#create` will result in the registry being built, registered to the `RegistryManager`, and returned to the caller for additional processing.
 
 The `Class` of the value of the registry must implement `IForgeRegistryEntry`, which defines that `#setRegistryName` and `#getRegistryName` can be called on the objects of that class. It is recommended to extend `ForgeRegistryEntry`, the default implementation instead of implementing the interface directly. When `#setRegistryName(String)` is called with a string, and that string does not have an explicit namespace, its namespace will be set to the current modid.
 
 Any newly created registry should use its associated [registration method][registration] to register the associated objects.
 
-### Using RegistryEvent$NewRegistry
+### Using NewRegistryEvent
 
-When using `RegistryEvent$NewRegistry`, all one needs to do is construct the registry within the event via `RegistryBuilder`. Since the registry is created after calling `#create`, the resulting registry can be stored in any `IForgeRegistry` field for future use.
+When using `NewRegistryEvent`, all one needs to do is construct the registry within the event via `#create`. The supplied registry can be accessed after `NewRegistryEvent` has finished posting to the mod event bus.
```

Comment:
This should more explicitly state that the returned value is a supplier of the IForgeRegistry.